convert2rhel/toolopts.py

```diff
@@ -199,10 +199,14 @@ def _register_options(self):
         group.add_option(
             "-c",
             "--config-file",
-            help="A configuration file to safely provide either a user password or an activation key for registering"
-            " the system through subscription-manager. Alternatively, passing these values through the"
-            " --activationkey or --password option would leak them through a list of running processes."
-            " Example of this file in /etc/convert2rhel.ini",
+            help="The configuration file is an optional way to safely pass either a user password or an activation key"
+            " to the subscription-manager to register the system. This is more secure than passing these values"
+            " through the --activationkey or --password option, which might leak the values"
+            " through a list of running processes."
+            " You can edit the pre-installed configuration file template at /etc/convert2rhel.ini or create a new"
+            " configuration file at ~/.convert2rhel.ini. The convert2rhel utility loads the configuration from either"
```

Comment:
Not sure if we want to actually write instead:

```suggestion
            " configuration file at ~/.convert2rhel.ini. The Convert2RHEL utility loads the configuration from either"
```

Reply:
That is for the product offering, we use all lowercase for the tool itself